libpod/container_internal_linux.go

```diff
@@ -373,6 +373,14 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 		return nil, err
 	}
 
+	if err := c.makeRunHostMount(g); err != nil {
+		return nil, err
+	}
+
+	if err := c.injectContainerUUID(g); err != nil {
+		return nil, err
+	}
```

Comment:
I think this should only be added for systemd containers

Reply:
I probably agree, does machinectl and other tools do anything in the case where systemd is not running within the container.

Reply:
We have some container information in /run/.containerenv, but we only leak information their in --privileged mode.

```
podman run --privileged fedora cat /run/.containerenv
engine="podman-4.0.0-rc4"
name="musing_swartz"
id="fde1c78d6d4cc8a461dc2eb53d6f235946b79e43f812a048b7c79ea5a71df950"
image="registry.fedoraproject.org/fedora:latest"
imageid="e417cd49a84e1749071c516c4f0013ea62113cb5adc98a8504a63a04bfd43479"
rootless=1
```